### PR TITLE
Add rhel dependencies and install instructions

### PR DESCRIPTION
**Pull request by Shelnutt2**
This adds the relevant dependencies needed for installing on RHEL(7) or Centos.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +47,15 @@ cat requirements.system | xargs sudo apt-get install -y
 sudo pip install -r requirements.txt
 ```
 
+#### RHEL or Centos
+```sh
+# Install apt-get dependencies.
```
Comment (josephschorr) [inline README.md]: Fix comment

**Comment by josephschorr** [comment]
@Shelnutt2: Should I re-review this?

**Comment by Shelnutt2** [comment]
@josephschorr Yes this is ready for re-review.

**Comment by josephschorr** [comment]
LGTM

**Pull request merged**